spec/password.spec.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+describe("password", ()  => {
+    var password, crypto;
+
+    beforeEach(() => {
+        var config;
+
+        crypto = require("crypto");
+        config = {
+            password: {
+                hashAlgo: "sha256"
+            }
+        };
+        password = require("../lib/password")(config, crypto);
+    });
+    it("hashes a passed in value", () => {
+        expect(password.hashContent("rRTcBER_EiFUsRa34Hj5Zpok")).not.toBe("rRTcBER_EiFUsRa34Hj5Zpok");
```

Comment:
What should it be?  I figure that would be a more accurate test.  Perhaps the code always returns `null`, in which case the test would pass and you'd get coverage, but it would not be a meaningful test.